tests/backup/backup_helper.go

```diff
@@ -4427,3 +4430,86 @@ func UpdateCluster(clusterName string, clusterUid string, kubeConfigPath string,
 	}
 	return status, err
 }
+
+// CreateRole creates role with given services and apis in px-backup datastore and also add role to keycloak.
+func CreateRole(roleName backup.PxBackupRole, svcs, apis []string, ctx context.Context) error {
```

Comment:
Can we convert the services and apis into types and accept only those types in this function rather than accepting string? This will make the test case also more readable.

Reply:
Done.